dev/com.ibm.ws.artifact.overlay/resources/com/ibm/ws/artifact/overlay/internal/resources/ArtifactOverlayMessages.nlsprops

```diff
@@ -0,0 +1,53 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.artifact.overlay/resources/com/ibm/ws/artifact/overlay/internal/resources/ArtifactOverlayMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKM
+#COMPONENTNAMEFOR CWWKM Module API
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# # A file path.
+overlay.does.not.exist=CWWKM0301E: The overlay folder {0} does not exist.
+overlay.does.not.exist.explanation=The liberty file system is not consistent with the overlay configuration.  This can occur because the server was restarted with changes to the overlay configuration.
+overlay.does.not.exist.useraction=Restart the server with the '--clean' option.
+
+# # A file path.
+overlay.not.directory=CWWKM0302E: The overlay folder {0} is not a directory.
+overlay.not.directory.explanation=The liberty file system is not consistent with the overlay configuration.  This can occur because the server was restarted with changes to the overlay configuration.
```

Comment:
Use "Liberty" instead of "liberty" unless all lowercase is needed. (global change)

Reply:
Fixed.